README.md

```diff
@@ -1,2 +1,26 @@
-# Animated-navbar
-Whathevs
\ No newline at end of file
+# Toolbar icons animation challenge
+<p align="center">
+  <img width="800" src="./images/Toolbar icons animation.gif"/>
+</p>
+
+## Description
+The aim of this project is to practice HTML, CSS and Vanilla JS by building a toolbar icons animation.
+This challenge helps realizing how we can build sophisticated things with simple code.
+
+## Installation
+1. Clone this repository from GitHub
+2. Open index.html in your browser.
+
+## Demo
+Willing to play with the toolbar? Click on the link below :point_down: 
+
+  https://toolbar-animation-icon.netlify.app/
+
+## Technologies used
+- HTML
+- CSS
+- Vanilla JS
```

Comment:
If you want, you could create a section `## Learnings` with a summary of your major learnings with this challenge ;)